### PR TITLE
Add profile for niall bambury and blog about code reviews

### PR DESCRIPTION
**Pull request by niall-rb**
This is a pull request for adding my profile and first blog to the site.

I've followed the contribution guide closely but if I've missed anything or gotten anything wrong then please let me know. 

I'm also happy to get feedback about the blog post itself if anyone is so inclined 🙏

### CODE REVIEW

**Review by x4d3 — APPROVED** [review]
Excellent article. Thank you !

Lot of good stuff that resonates to me.

**Review by pataruco — APPROVED** [review]

File: _posts/2022-06-22-human-centered-code-reviews.md [hunk 1]
```diff
@@ -0,0 +1,212 @@
...
+
+Today’s devs can even provision production-like services on their development machines through techniques like containerisation, providing lean standalone services - the kind of thing that would have been unimaginable for complicated subsystem teams of the past.
+
+For companies that have leaned into continuous delivery and continuous integration as practices, code deployments that once took hours or days could be achieved in mere minutes. Tools like Github Actions can run automated checks against our bundled software before deploying it to production. Some of these checks can include automated tests (unit, e2e, contract etc..), static analysis tools, vulnerability scans just to name a few.
+
+So what does this all mean? 
+
+The need for a second pair of eyes notwithstanding, the code review has gone from being one of a handful of practices to ensure code quality to just one among many. And even at that, it’s prime responsibility of ensuring more than one dev has seen the code is satisfied by the emergency of another rival practice - pair programming.
```
Comment (IkeLutra) [inline _posts/2022-06-22-human-centered-code-reviews.md:55]:
```suggestion
The need for a second pair of eyes notwithstanding, the code review has gone from being one of a handful of practices to ensure code quality to just one among many. And even at that, it’s prime responsibility of ensuring more than one dev has seen the code is satisfied by the emergence of another rival practice - pair programming.
```

Is that what you meant? The last sentence is a little tricky to parse for me so maybe reword it?

**Review by IkeLutra — COMMENTED** [review]
This was a really interesting read thanks for writing. I think I spotted one typo and have commented it.

File: _posts/2022-06-22-human-centered-code-reviews.md (hunk 1 above)
Comment (niall-rb) [reply _posts/2022-06-22-human-centered-code-reviews.md:55]: Good spot Will! I'll cover off that typo and see if I can make the sentence a bit more readable

**Pull request merged**